R/pkg/R/DataFrame.R

```diff
@@ -2685,7 +2686,8 @@ setMethod("unionAll",
 
 #' Union two or more SparkDataFrames
 #'
-#' Union two or more SparkDataFrames. This is equivalent to \code{UNION ALL} in SQL.
+#' Union two or more SparkDataFrames by row. In constrast to \link{union}, this method
+#' requires that the input SparkDataFrames have the same column names.
```

Comment:
Thanks. Updated doc. R's `rbind` seems to do type conversion similarly to `union`:
```
df <- data.frame(name = c("Michael", "Andy", "Justin"), age = c(1, 30, 19))
df2 <- df
df2$age <- as.character(df2$age)
rbind(df, df2)
     name age
1 Michael   1
2    Andy  30
3  Justin  19
4 Michael   1
5    Andy  30
6  Justin  19
str(rbind(df, df2))
'data.frame':	6 obs. of  2 variables:
 $ name: Factor w/ 3 levels "Andy","Justin",..: 3 1 2 3 1 2
 $ age : chr  "1" "30" "19" "1" ...
```